core/src/test/java/org/apache/brooklyn/core/mgmt/rebind/RebindTestUtils.java

```diff
@@ -238,7 +238,6 @@ public LocalManagementContext buildUnstarted() {
                     classLoader);
             ((RebindManagerImpl) unstarted.getRebindManager()).setPeriodicPersistPeriod(persistPeriod);
             unstarted.getRebindManager().setPersister(newPersister, PersistenceExceptionHandlerImpl.builder().build());
-            unstarted.getHighAvailabilityManager().disabled();
```

Comment:
I added this recently in https://github.com/apache/brooklyn-server/pull/607, but no longer needed after the changes in https://github.com/apache/brooklyn-server/pull/617 which make the HA state configurable.